solr/core/src/test/org/apache/solr/search/TestFilteredDocIdSet.java

```diff
@@ -127,22 +130,49 @@ public void testNullDocIdSet() throws Exception {
     IndexSearcher searcher = newSearcher(reader);
     Assert.assertEquals(1, searcher.search(new MatchAllDocsQuery(), 10).totalHits.value);
     
-    // Now search w/ a Filter which returns a null DocIdSet
-    Filter f = new Filter() {
+    // Now search w/ a Query which returns a null DocIdSet
+    Query f = new Query() {
```

Comment:
This test should be testing DocSetQuery.  If it's a null set of IDs, then the underlying DocIdSetIterator will be null.

Reply:
Thanks for the catch. I just had that corrected!